CI-CD/README.md

```diff
@@ -28,7 +28,7 @@ See also: [Cache the Latest Version of the Unified Agent](../Scripts/README.md#c
 
 ## Pipeline Log Publishing
 
-* Publish the `whitesource` folder with logs & reports by adding the following commands depending on each pipeline
+* Publish the `Mend` folder with logs & reports by adding the following commands depending on each pipeline
```

Comment:
Same as before, is this folder name supported? Is it the (still) hard-coded ./whitesource folder the UA creates, or is this an arbitrary name in this context?

Reply:
@lukebrogan-mend this should remain /whitesource as it is the folder for the unified agent scan...which I believe is still whitesource